.github/workflows/syft-security.yml

```diff
@@ -60,13 +60,8 @@ jobs:
       - name: Install Dependencies
         if: steps.changes.outputs.syft == 'true'
         run: |
-          pip install packaging --default-timeout=60
```

Comment:
Unfortunately we need to install all the packages or the safety check doesn't work.

```
$ safety check
safety report
checked 24 packages, using free DB (updated once a month)
---
-> pillow, installed 8.2.0, affected <=8.2.0, id 40965

$ echo $?
255
 
$ pip uninstall pillow
Found existing installation: Pillow 8.2.0
Uninstalling Pillow-8.2.0
  Successfully uninstalled Pillow-8.2.0

$ safety check
safety report
checked 23 packages, using free DB (updated once a month)
---
No known security vulnerabilities found.

```

Reply:
Reverted Back